host/cli/bootstrap.go

```diff
@@ -296,6 +296,23 @@ $$;`, step.Artifact.URI, step.ID))
 		data.MongoDB.ImageArtifact.URI = artifactURIs["mongodb"]
 	}
 
+	// set TELEMETRY_CLUSTER_ID
+	telemetryClusterID := random.UUID()
+	sqlBuf.WriteString(fmt.Sprintf(`
+UPDATE releases SET env = jsonb_set(env, '{TELEMETRY_CLUSTER_ID}', '%q')
+WHERE release_id = (SELECT release_id FROM apps WHERE name = 'controller');
```

Comment:
Is the intention to explicitly change this value when restoring from a backup, or should there be a `WHERE env->>'TELEMETRY_CLUSTER_ID' IS NULL` check?